ulm-semantics/main/hooks/bytes.md

```diff
@@ -345,6 +345,7 @@ module ULM-SEMANTICS-HOOKS-BYTES
 
     rule ulmBytesHash(ptrValue(_, u64(BytesId)))
         => ulmBytesHash(ulmBytesId(BytesId))
+    // TODO: use Keccack here.
```

Comment:
Why is this needed?

Reply:
Strictly speaking, we don't need keccack here. What we actually want is:
* A hash function that has a low collision probability
* Using all 256 bits of the hash.

It's just that we are already using keccack in other places so we wouldn't add new dependencies or complexity by using it.

Reply:
I rewrote the TODO.